service-api/app/src/App/src/Handler/PaperVerification/PaperVerificationValidationHandler.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Handler\PaperVerification;
+
+use App\Exception\ApiException;
+use App\Exception\BadRequestException;
+use App\Exception\GoneException;
+use App\Exception\NotFoundException;
+use App\Handler\Trait\RequestAsObjectTrait;
+use App\Request\PaperVerificationCodeValidate;
+use App\Request\ViewerCodeFull;
+use App\Service\Lpa\LpaManagerInterface;
+use App\Service\PaperVerificationCodes\PaperVerificationCodeService;
+use Laminas\Diactoros\Response\JsonResponse;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+
+/**
+ * @codeCoverageIgnore
+ */
+class PaperVerificationValidationHandler implements RequestHandlerInterface
+{
+    /** @use RequestAsObjectTrait<ViewerCodeFull> */
+    use RequestAsObjectTrait;
+
+    public function __construct(
+        private LpaManagerInterface $lpaManager,
+        private PaperVerificationCodeService $paperVerificationCodeService,
```

Comment:
```suggestion
        private PaperVerificationCodeService $codeService,
```

Just to keep it all in line